application/forms.py

```diff
@@ -8,26 +8,26 @@ class SignUpForm(FlaskForm):
     Class to encapsulate sign up form
     """
     first_name = StringField(
-        'First Name:',
+        'First name',
```

Comment:
In the form created by `wtf-forms` do we ant space separated keys?

Reply:
We might, but with the current UI there is no need since it looks like:

```
First name
---------------------------------
|                                                      |
---------------------------------
```

Have you launched this branch? Check it out and let me know. This is largely still a WIP PR.

Reply:
I haven't gotten to system test yet - will get to that later though